### ISSUE TITLE
Add an example of importing a CSV to the tuning cookbook

### ISSUE THREAD

**Issue by pcoet** [google-gemini/cookbook#293]
### Description of the feature request:

Add an example of importing a CSV using `pandas.read_csv` to the tuning cookbook: https://github.com/google-gemini/cookbook/blob/main/quickstarts/Tuning.ipynb

AI Studio accepts csv files as datasets for tuning jobs. The tuning cookbook currently demonstrates creating a tuned model using an list of dictionaries. This is a request for an additional sample demonstrating importing a training dataset from a local csv file using `pandas.read_csv`, to mirror the functionality of importing a csv file through AI Studio.

### What problem are you trying to solve with this feature?

Help users understand how to tune a model using data from a CSV.

**Comment by pcoet** [comment]
This could also be a new notebook based on the existing tuning notebook.

**Comment by markmcd** [comment]
I don't think this necessarily needs a new guide, at least not for Python. `genai.create_tuned_model()` can take a CSV file handle or string filename as the `training_data` arg.

So this
```py
genai.create_tuned_model(
    training_data=[
        { ... }, { ... }, etc
    ], ...
)
```

becomes
```py
genai.create_tuned_model(
    training_data='my_file.csv', ...
)
```

It supports JSON files too.

**Comment by Qamar2315** [comment]
hi

**Comment by markmcd** [comment]
Fixed in #300
